test/manual/atlas-data-lake-testing/atlas_data_lake_testing.prose.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+const { expect } = require('chai');
+const { withClient } = require('../../integration/shared');
+
+// TODO: These tests are not fully implemented per the spec
```

Comment:
Do we want a separate ticket for fixing this?

Reply:
I think the best approach would be yes lets file an improvement ticket and keep sorting through our compliance